apps/arena/lib/arena/matchmaking.ex

```diff
@@ -7,4 +7,22 @@ defmodule Arena.Matchmaking do
   def get_queue("quick-game"), do: Arena.Matchmaking.QuickGameMode
   def get_queue("deathmatch"), do: Arena.Matchmaking.DeathmatchMode
   def get_queue(:undefined), do: Arena.Matchmaking.GameLauncher
+
+  def get_matchmaking_configuration(matchmaking_state, team_size, game_mode_type) do
+    if Map.has_key?(matchmaking_state, :game_mode_configuration) do
+      matchmaking_state
+    else
+      case Arena.Configuration.get_game_mode_configuration(team_size, game_mode_type) do
+        {:error, _} ->
+          matchmaking_state
+
+        {:ok, game_mode_configuration} ->
+          # This is needed because we might not want to send a request every 300 seconds to the game backend
```

Comment:
You forgot to delete this comment right?

Reply:
yes!